src/backend/gitporter/gitprovider/__init__.py

```diff
@@ -17,9 +18,9 @@
 }
 
 
-def getRepositoryList(platform: GitPlatform, workspace: str):
+def get_remote_repositories(platform: GitPlatform, workspace: str) -> t.List[RemoteRepositoryInformation]:
```

Comment:
```suggestion
def get_remote_repositories_infos(platform: GitPlatform, workspace: str) -> t.List[RemoteRepositoryInformation]:
```
doesn't sound good, but better than a misleading name

Reply:
Ehrlich gesagt verwirrt mich das andere viel mehr wegen der falschen Grammatik. Wenn man genauer wissen will, was da zurückkommt, hat man ja den Type. Deswegen habe ich es im type extra explizit hingeschrieben, was das ist.